### PR TITLE
Update jellyfin.md

### PR DESCRIPTION
**Pull request by ThorpeJosh**
Great project. Many thanks to the maintainers.

I have added some missing fields and restructured to match the jellyfin plugin UI.

### CODE REVIEW

File: example_configs/jellyfin.md [hunk 1]
```diff
@@ -1,42 +1,47 @@
...
 ```
-(memberof=cn=media,ou=groups,dc=example,dc=com)
+uid
 ```
-
-Otherwise, just use:
+### LDAP Password Attribute
 ```
-(uid=*)
+userPassword
```
Comment (nitnelave) [inline example_configs/jellyfin.md:41]: Is that the main difference? It's a bit hard to tell given that the fields were reordered. 

Anyway, this field is not, will not and cannot be available through LLDAP by design.

**Review by nitnelave — CHANGES_REQUESTED** [review]
I'm all for a little cleaning up, but the userPassword field cannot work. Have you tried it?

File: example_configs/jellyfin.md (hunk 1 above)
Comment (ThorpeJosh) [reply example_configs/jellyfin.md:41]: I agree it is difficult to see the diff. I recommend just looking at the entire rendered file side by side.

90% of changes is just reordering and adding some `## Headings` to match the Jellyfin UI sections.
The only modification to existing was to change the default bind user from `admin` to `jellyfin_bind_user`.
The only addition was adding the following block:
```markdown
### LDAP Password Attribute
\```
userPassword
\```
```

The reason I added it in the pr is:
- It was the only attribute from jellyfin UI that wasn't listed in `jellyfin.md`
- `userPassword` is the default jellyfin value and it is functioning as expected for me.
- It is mentioned to at the bottom of `jellyfin.md` under the `password change` section:
```markdown
`LDAP Password Attribute` Needs to be set to `userPassword`
```

Comment (ThorpeJosh) [reply example_configs/jellyfin.md:41]: @nitnelave Given that `userPassword` is not available in lldap, did you want me to remove my addition and the last line of the `Password change` section introduced in c440df6 ?

Comment (nitnelave) [reply example_configs/jellyfin.md:41]: The last line is fine: you can change passwords through jellyfin, but you can't read them (which the attribute is about) 

So yes, please remove the `userPassword` section, the rest of the PR looks fine

Comment (ThorpeJosh) [reply example_configs/jellyfin.md]: I've removed the requested section. Let me know if anything else needs changing.

**Review by nitnelave — APPROVED** [review]
Thanks, LGTM !

**Review by nitnelave — APPROVED** [review]

**Pull request merged**